README.md

```diff
@@ -15,3 +15,8 @@ https://www.loginradius.com/docs/developer
 -  Open a browser to the link provided in the console.
 
 
+## How to contribute
+
+We appreciate all kinds of contributions from anyone, be it finding an issue or writing a blog.
```

Comment:
Change it to `We appreciate all kinds of contributions from our contributors.`

Reply:
Just updated check now.